internal/cluster/distribution/eks/eksprovider/workflow/select_volume_size_activity.go

```diff
@@ -70,7 +70,7 @@ func (a *SelectVolumeSizeActivity) Execute(ctx context.Context, input SelectVolu
 
 	if output.VolumeSize < input.AMISize {
 		return nil, errors.New(fmt.Sprintf(
-			"selected volume size of %d GB (source: %s) is less than the AMI size of %d GB",
+			"selected volume size of %d GiB (source: %s) is less than the AMI size of %d GiB",
```

Comment:
**Note**: we need to add this to the release changes as a patch, because people could be depending on the returned error string (even though not advised) which changes here, so technically there is a (breaking) behavior change with this PR (but not a breaking API change which is the main thing tracked in the semantic versioning). (This is more of a self note for release than a review comment, but the thought process is worth sharing.)

If this would be a named/marked/highlighted/individualized/interfaced error (which is encouraged to be built upon by downstream users) then I would even put it into the BREAKING CHANGES release category.
(But if implicit behavior change would cause BREAKING CHANGES then almost every release would be a BREAKING CHANGE which 95% of the time wouldn't require code modifications from downstream users which would be noisy and would defeat the purpose of using the BREAKING CHANGES indicator., so that's why I'm saying it is fine to be a patch IMO - everyone's flavor may differ.)